common/src/main/java/com/wynntils/screens/guides/WynntilsGuideScreen.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.guides;
+
+import com.wynntils.screens.base.WynntilsListScreen;
+import com.wynntils.screens.base.widgets.BackButton;
+import com.wynntils.screens.base.widgets.ItemSearchWidget;
+import com.wynntils.screens.base.widgets.PageSelectorButton;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.services.itemfilter.SearchQuery;
+import com.wynntils.utils.render.Texture;
+import net.minecraft.network.chat.Component;
+
+public abstract class WynntilsGuideScreen<E, B extends WynntilsButton> extends WynntilsListScreen<E, B> {
+    protected WynntilsGuideScreen(Component component) {
+        super(component);
+        this.searchWidget =
+                new ItemSearchWidget(0, -22, Texture.QUEST_BOOK_BACKGROUND.width(), 20, q -> reloadElements(), this);
+    }
+
+    @Override
+    protected void doInit() {
+        super.doInit();
+
+        // TODO Add the explanation text
```

Comment:
What kind of explanation?

Reply:
Something listing the available filters, but there is not much space so we must think of something. It might be in the same format as legacy for example : https://user-images.githubusercontent.com/3767283/131205683-05a060ba-23db-4ae4-8004-f1d1500034fe.png

Reply:
I think it might be acceptable to skip the explanation for now. This PR has been open long enough; it is now of sufficiently high quality to be merged imho, so then it's better to merge it and fix additions as follow up PRs.